Misc/NEWS.d/next/IDLE/2021-05-28-09-28-45.bpo-43654.WYsKKG.rst

```diff
@@ -0,0 +1,5 @@
+Fix the config dialog breaking tab completion.  This is achieved by removing
+the key configuration options for ``<<smart-indent>>`` (Tab),
+``<<newline-and-indent>>`` (Return, Enter) and ``<<smart-backspace>>``
+(Backspace) from the configuration mechanism, making them hard-coded
+instead.
```

Comment:
These two are not relevant to tab completion.  So it we do these together, I think we need to change the PR title (see my  issue title revision) and reword this a bit.

Reply:
I agree.

Reply:
I'm not sure the PR/commit title must describe all of the technical changes made. The additional change to the `<<newline-and-indent>>` and `<<smart-backspace>>` bindings has negligible significance to users, since even before this change, changing these bindings breaks IDLE rather badly.

Having this detail in the NEWS entry, and possibly in the body of the commit message, seems right to me.

Reply:
To be clear, I now thing that the existing PR title is good, since it describes the only significant bug being fixed.

Reply:
Today I think the title is fine as is.